File: 1-Iniciante/Ex1060.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Ex1060 { // para ser aceito no Beecrowd mude o nome da classe para Main
    public static void main(String[] args) throws IOException {
        // variaveis & objetos
        BufferedReader bf = new BufferedReader(new InputStreamReader(System.in));

        // entrada & processamento
        int contPositivos = 0;
        for (int i = 0; i < 6; i++) {
            if (Double.parseDouble(bf.readLine()) > 0d) {
                contPositivos++;
            }
        }
        bf.close();
        // saida
        System.out.printf("%d valores positivos\n", contPositivos);
    }
}

File: 1-Iniciante/ex1060.js
// entrada
let input = require('fs').readFileSync('dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd

// processamento
let cont_positivos = 0;
input.split('\n').map(v => {
    let x = parseFloat(v);
    if(x > 0.0) cont_positivos++;
});

// saida
console.log(cont_positivos + ' valores positivos');